app_server/app/adapters/socket/Client.js

```diff
@@ -17,7 +17,7 @@ function Client(socket, credentials) {
 
   this.socket = socket;
   this.credentials = credentials;
-  this.rooms = {};
+  this.rooms = {}; // A map from room's name to Room object
```

Comment:
There could be a room for notification and stuffs. Either way, this allows more flexibilities in the future such as viewing multiple streams on the same app etc. without affecting our current simpler feature.